structopt-derive/src/parse.rs

```diff
@@ -18,11 +18,10 @@ pub struct StructOptAttributes {
 impl Parse for StructOptAttributes {
     fn parse(input: ParseStream<'_>) -> syn::Result<Self> {
         let content;
+        let paren_token = parenthesized!(content in input);
+        let attrs = content.parse_terminated(StructOptAttr::parse)?;
 
-        Ok(StructOptAttributes {
-            paren_token: parenthesized!(content in input),
-            attrs: content.parse_terminated(StructOptAttr::parse)?,
-        })
+        Ok(StructOptAttributes { paren_token, attrs })
```

Comment:
Why clippy doesn't like that?

Reply:
Because initialization order is not really defined (yet). De-facto it is left-to-right but relying on this is relying on unspecified behavior.

Reply:
OK, thanks, good to know, I use that sometime.

Reply:
I didn't know about it either until clippy told me :) The more you know and all...